src/main/java/io/odh/test/platform/KubeUtils.java

```diff
@@ -64,4 +68,29 @@ public static void waitForInstallPlan(String namespace, String csvName) {
 
     private KubeUtils() {
     }
+
+    @io.qameta.allure.Step
```

Comment:
well why is the allure step is in kubeutils?

Reply:
because it used to be allure step when it was private method duplicated into three different test classes; I wanted to keep it as allure step after moving it to utils;

I can remove this if you prefer; and I can undo the pom.xml change where I made allure available in not-tests.

Reply:
I don't want to have allure steps in utils and main part

Reply:
ack!